src/ServiceControl.Audit.AcceptanceTests.RavenDB/ServiceControl.Audit.AcceptanceTests.RavenDB.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net472</TargetFramework>
+    <RunSettingsFilePath>ServiceControl.runsettings</RunSettingsFilePath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="../ServiceControl.Audit.AcceptanceTests/**/*.cs"
```

Comment:
If we move all of the shared tests into a subfolder theen we can do without the excludes. This might be a bigger piece of work and make history harder to follow so maybe it's not worth it.

Reply:
thought about that as well but called YAGNI for now, lets tackle this once we go through all acceptance tests and decides which are better covered as a persistence test